docs-src/main/userguide/tables.md

```diff
@@ -171,8 +171,36 @@ Tablesaw supports inner and outer joins between tables.
 
 
 ## Add and remove rows
+###Adding a new Row
 
+To add a new row into the table you simply need to call
+```java
+Row newRow = aTable.appendRow();
+```
+then just set the values of each column in the row
+```java
+newRow.setString("s","value1");
+newRow.setDate("d",LocalDate.now());
+```
+
+###Remove rows from table
+
+To remove rows from table you need to call *dropRows()* on table with indexes that you want to drop
```

Comment:
A bit wordy. Please change to something like.

To remove rows call *dropRows()* with indexes of the rows you want to drop.

Reply:
Yeah sure